src/internal/pfsdb/repos_test.go

```diff
@@ -299,3 +300,39 @@ func assertRepoSequence(t *testing.T, names []string, repos []pfsdb.RepoInfoWith
 		require.Equal(t, n, repos[i].RepoInfo.Repo.Name)
 	}
 }
+
+func testRepoPicker() *pfs.RepoPicker {
+	return &pfs.RepoPicker{
+		Picker: &pfs.RepoPicker_Name{
+			Name: &pfs.RepoPicker_RepoName{
+				Project: testProjectPicker(),
+				Name:    testRepoName,
+				Type:    testRepoType,
+			},
+		},
+	}
+}
+
+func TestPickRepo(t *testing.T) {
```

Comment:
This might be a good place to add a table test.  I see there is one case above.  If that's in a table it makes it obvious how to extend the test.